PublicationsRouter/v2/api/API.md

```diff
@@ -585,3 +586,52 @@ If the notification content is found and authentication succeeds you will receiv
     Binary Package
 
 Note that a successful access by a Repository account user will log a successful delivery of content notification into PubRouter (used for reporting on PubRouter's ability to support REF compliance).
+
+# API for matching configuration
+If you are a repository user you have access to the endpoint `/config`. This endpoint allows you to either update or retrieve your current matching configuration. This endpoint requires authentication, using an api key. 
+
+## Retrieving Matching Configuration
+
+To retrieve your current matching configuration from this endpoint, simply make a GET request against the endpoint with your API key. 
+
+    GET /config?api_key=<api_key>
+
+If authentication was successful then you will receive a 200 (OK) and a JSON containing your current repository configuration:
+
+    HTTP 1.1  200 OK
+    Content-Type: application/json
+    
+    {
+        "matching_configuration": ["values"],
+        ...
+    }
+
+For information regarding the JSON data structure returned, see [matching config structure](./MatchingConfig.md).
+
+## Submitting Matching Configuration
+
+To submit new matching configuration changes using this endpoint, you must make a POST request against the endpoint. You have two choices with regards to the format of content you submit. You can either submit a JSON data structure, or a csv file. 
```

Comment:
Need to make clear that if they submit new matching  config it always REPLACES what previously existed.

`To submit new matching configuration you make a POST request to this endpoint.  You may submit the changes either as a JSON data structure or as a CSV file.`